packages/acter_notifify/lib/processing/attachment_data_process.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:acter_notifify/model/push_styles.dart';
+import 'package:acter_notifify/processing/parent_data_process.dart';
+
+(String, String?) titleAndBodyForAttachment(NotificationItem notification) {
+  //Generate attachment body
+  final sender = notification.sender();
+  final username = sender.displayName() ?? sender.userId();
+  final attachmentTitle = notification.title();
+
+  String? body =
+      "$username added ${PushStyles.attachment.emoji} $attachmentTitle";
+
+  //Generate attachment title
+  final parent = notification.parent();
+  String title;
+  if (parent != null) {
+    final parentInfo = parentPart(parent);
+    title = parentInfo;
+  } else {
+    title = body;
+    body = null;
+  }
+
+  return (title, body);
```

Comment:
Can't this be nicely shortened to...

```suggestion
  final content =
      "$username added ${PushStyles.attachment.emoji} $attachmentTitle";

  //Generate attachment title
  final parent = notification.parent();
  if (parent != null) {
    final parentInfo = parentPart(parent);
    return (parentInfo, content);
  } else {
    return (content, null);
  }
```